HW_7/HW_7_Task1.py

```diff
@@ -0,0 +1,25 @@
+# Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в его стихах
+# ритм. Поскольку разобраться в его кричалках не настолько просто,
+# насколько легко он их придумывает, Вам стоит написать программу.
+# Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе
+# стихотворения одинаковое. Фраза может состоять из одного слова, если во
+# фразе несколько слов, то они разделяются дефисами. Фразы отделяются друг
+# от друга пробелами. Стихотворение  Винни-Пух вбивает в программу с
+# клавиатуры. В ответе напишите “Парам пам-пам”, если с ритмом все в
+# порядке и “Пам парам”, если с ритмом все не в порядке
+
+# *Пример:*
+
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да
+#     **Вывод:** Парам пам-пам
+
+def word(text):
+    str = text.lower().split()
+    def x(x): return sum(1 for i in x if i in "аоуыэеёиюя")
```

Comment:
Запись нехарактерная для Python.  Это не является ошибкой, просто так не принято использовать.
```suggestion
    def x(x):
        return sum(1 for i in x if i in "аоуыэеёиюя")
```

Reply:
Вообще, объявляя функцию внутри другой функции, вы теряете к ней доступ из глобальной области видимости, а если вы ее определите на одном уровне с внешней функцией, то функциональность внешней функции никак не изменится